### PR TITLE
Milestone: Setup

### PR DESCRIPTION
**Pull request by Roseokpe**
I created the  foundations for the Bookstore website including the router and react part.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @Roseokpe, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them with `npx stylelint --fix "**/*.{css,scss}"`, but please get used to fixing the errors line by line also because it'll help you learn best practices.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```
Comment (ahmetbozaci) [inline README.md]: - [ ] Kindly update your readme file

-You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections. 
You should change the readme template's sentence with your sentence.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Roseokpe  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
 ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***
- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by descholar-ceo — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**